x/tokenfactory/keeper/admins.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/desmos-labs/desmos/v5/x/tokenfactory/types"
+)
+
+// GetAuthorityMetadata returns the authority metadata for a specific denom
+func (k Keeper) GetAuthorityMetadata(ctx sdk.Context, denom string) types.DenomAuthorityMetadata {
+	bz := k.GetDenomPrefixStore(ctx, denom).Get([]byte(types.DenomAuthorityMetadataKey))
+
+	metadata := types.DenomAuthorityMetadata{}
```

Comment:
I think this can be simply declared without initializing it 
```suggestion
	var metadata types.DenomAuthorityMetadata
```